app/CreateDeal.tsx

```diff
@@ -60,6 +63,21 @@ export function CreateDeal(props: {
                 />
               </div>
             </FormGroup>
+            <FormGroup>
+              <FormLabel>Unit</FormLabel>
+              {units.map((unit, index) => {
+                return (
+                  <FormCheckRadio
+                    key={index}
+                    onChange={(newUnit: string) => {
+                      setSelectedUnit(newUnit);
+                    }}
+                    unit={unit}
+                    value={selectedUnit}
+                  ></FormCheckRadio>
```

Comment:
empty tags should be self-closing

Reply:
https://github.com/auberginearms/groceryDatabaseApp/pull/30/commits/733a416f371b65065993184c50e63f296b41b896